Member-Travel-Fund.md

```diff
@@ -0,0 +1,40 @@
+## Purpose
+
+To establish and administer a fund for members of the Node.js
+Foundation to travel and spread knowledge about the foundation
+and the Node.js Foundation projects.
+
+## Restrictions
+
+* Candidates must be Individual Members of the Node.js Foundation.
+* Candidates must be presenting one of the following presentations:
+ * [The Node.js Foundation](https://github.com/nodejs/foundation-slides).
+
+## Process
+
+New slide decks are added to the list of approved decks on an ongoing basis.
+
+* Slide deck must be managed by the Node.js Evangelism WG.
+* Send a pull request to this file adding the slides to
+ the list above.
+* Slide deck must be approved by the TSC.
+
+Any member can apply to give an approved talk. Request may take up to a month
+for the TSC to approve or decline.
+
+* Log an issue in this repository.
+ * Include the name of the event you plan to attend.
+ * Include the location of the event and where you will be traveling from.
+ * Include the presentation you intend to give.
+ * Include the size of the stipend you wish to receive.
+  * Stipends are anywhere from $100 to $1,500 USD and vary with travel distance.
+
+The TSC will consider each proposal in its next regular meeting. While it is
+ultimately at the TSC's discretion who to approve the following considerations
+are made.
+
+* Impact: Preference given to underserved communities.
```

Comment:
> underserved instead of underrepresented

They can be separate but I'd say in this case underrepresented is either a subset of under-served or a by-product of being under-served. What we're trying to address is an information disparity and intervene by spreading that knowledge where we have the opportunity to do so. That disparity exists in several dimensions, some being geographic or language barriers and other being race or gender. Without this information being more accessible to these communities they are less represented because a prerequisite to being represented would be to contribute and these talks do a lot to show where and how to contribute.

Also, by "underserved" we mean not only that an information gap exists but that it is not "served" by the existing speakers already available to these events without our intervention.

In terms of this program, we can only address what we are presented with the opportunity to address (people have to apply to the program, and there are only so many opportunities to speak especially in areas we know are under-served).

However, we make sure the program isn't too specific so that it can address new issues as they arise. If, for instance, we want to create a deck about libuv targeted at getting more contributors involved in that project we may want to adjust the communities we target based on other criteria (existing Windows developers, for example).